docs/src/documentation/03-components/05-input/choicetile/01-guidelines.mdx

```diff
@@ -0,0 +1,15 @@
+---
+title: Guidelines
+redirect_from:
+  - /components/choicetile/
+---
+
+<ReactExample exampleId="ChoiceTile-default" />
+
+## When to use
+
+- To present a set of similar options to chose from. Either only one option or multiple.
```

Comment:
I think we need more info here, something like we have it with other components. When to use / When not to use. Ask designers for more information about usage.

Reply:
I asked already. I just added this here so we don't have a blank page